pkg/param/render.go

```diff
@@ -119,11 +124,25 @@ func renderStringArg(arg string, tp TemplateParams) (string, error) {
 	}
 	buf := bytes.NewBuffer(nil)
 	if err = t.Execute(buf, tp); err != nil {
+		// Check if Error is because of undefined key,
+		// which will lead on execute error on first undefined (missingkey=error).
+		// Get the undefined key name from the error message.
+		if strings.Contains(err.Error(), undefinedKeyErrorMsg) {
+			return "", newUndefinedKeyError(err.Error())
+		}
 		return "", errors.WithStack(err)
 	}
 	return buf.String(), nil
 }
 
+func newUndefinedKeyError(err string) error {
+	pos := strings.LastIndex(err, undefinedKeyErrorMsg)
+	adjustedPos := pos + len(undefinedKeyErrorMsg)
+	key := err[adjustedPos:]
+	key = strings.Trim(key, "\"")
+	return errors.WithStack(errors.New("Can't render template: key <" + key + "> Not found"))
```

Comment:
let's use `fmt.Sprintf` here.
I was thinking if we can say something like this `key` is not already set, or was not set by previous actions.

Reply:
```suggestion
	return errors.WithStack(errors.New("Can't render template: "key" is not set"))
```